application_sdk/constants.py

```diff
@@ -130,10 +130,20 @@
 STATE_STORE_NAME = os.getenv("STATE_STORE_NAME", "statestore")
 #: Name of the secret store component in DAPR
 SECRET_STORE_NAME = os.getenv("SECRET_STORE_NAME", "secretstore")
-#: Name of the object store component in DAPR
-OBJECT_STORE_NAME = os.getenv("OBJECT_STORE_NAME", "objectstore")
+#: Name of the deployment object store component in DAPR
+DEPLOYMENT_OBJECT_STORE_NAME = os.getenv("DEPLOYMENT_OBJECT_STORE_NAME", "objectstore")
+#: Name of the upstream object store component in DAPR
+UPSTREAM_OBJECT_STORE_NAME = os.getenv("UPSTREAM_OBJECT_STORE_NAME", "atlan-storage")
```

Comment:
default must be `objectstore` - the deployment environment must configure it to atlan-storage.